dashboard/test/controllers/pd/professional_learning_landing_controller_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class Pd::ProfessionalLearningLandingControllerTest < ::ActionController::TestCase
+  setup do
+    @csf_workshop = create :pd_workshop, num_sessions: 3, course: Pd::Workshop::COURSE_CSF, subject: nil
+    @csd_workshop = create :pd_workshop, num_sessions: 3, course: Pd::Workshop::COURSE_CSD, subject: nil
+    @csp_workshop = create :pd_workshop, num_sessions: 3, course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP
+
+    @teacher = create(:admin, email: 'test_email@foo.com', user_type: 'teacher')
```

Comment:
Then please add a test to validate that a normal teacher can't call it (`assert_response :forbidden`).